.vscode/settings.json

```diff
@@ -1,8 +1,87 @@
 {
-    "editor.formatOnSave": true,
-    "editor.formatOnPaste": false,
-    "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
-    },
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.formatOnSave": true,
+  "editor.formatOnPaste": false,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "files.associations": {
```

Comment:
What are these added lines for? Can we remove them if they're not necessary?

Reply:
I'm not sure where those came from, but I don't think they're necessary.

Reply:
Ok, please remove the added lines in another commit.